tomni/cytosmart_data_format/annotations/ellipse/main.py

```diff
@@ -1,42 +1,133 @@
-from cell_fea.morphology import circularity
+from dataclasses import asdict
+from typing import List
+
+import numpy as np
 
 from ..annotation import Annotation
 from ..point import Point
 
 
 class Ellipse(Annotation):
-    def __init__(self, radius: Point, center: Point, rotation: float):
+    def __init__(
+        self,
+        radius: Point,
```

Comment:
Replace with 2 parameters; radius x and radius y
Or with radius object

Reply:
Tom, bram and I were discussing this. Atm we only have `circles` that are ellipses but for future we should also consider how we use elllipses, so let's talk about how we ellipses.

Reply:
Let discuss it Wednesday